age/smart-contract/src/deploy.ts

```diff
@@ -2,45 +2,56 @@ import * as dotenv from 'dotenv';
 import { readFileSync } from 'fs';
 import path from 'path';
 import { fileURLToPath } from 'url';
-import {
-  deploySC,
-  WalletClient,
-  ISCData,
-  IDeploymentInfo,
-} from '@massalabs/massa-sc-deployer';
-import { Args, IEvent, fromMAS } from '@massalabs/massa-web3';
+import { deploySC, WalletClient, ISCData } from '@massalabs/massa-sc-deployer';
+import { Args, fromMAS } from '@massalabs/massa-web3';
 
+// Load .env file content into process.env
 dotenv.config();
 
+// Get the URL for a public JSON RPC API endpoint from the environment variables
 const publicApi = process.env.JSON_RPC_URL_PUBLIC;
 if (!publicApi) {
   throw new Error('Missing JSON_RPC_URL_PUBLIC in .env file');
 }
-const privKey = process.env.WALLET_PRIVATE_KEY;
-if (!privKey) {
+
+// Get the secret key for the wallet to be used for the deployment from the environment variables
+const secretKey = process.env.WALLET_PRIVATE_KEY;
+if (!secretKey) {
   throw new Error('Missing WALLET_PRIVATE_KEY in .env file');
 }
 
-const deployerAccount = await WalletClient.getAccountFromSecretKey(privKey);
+// Create an account using the private key
+const deployerAccount = await WalletClient.getAccountFromSecretKey(secretKey);
 
+// Obtain the current file name and directory paths
 const __filename = fileURLToPath(import.meta.url);
-
 const __dirname = path.dirname(path.dirname(__filename));
 
+/**
+ * Deploy one or more smart contracts.
+ *
+ * @remarks
+ * Multiple smart contracts can be deployed by adding more objects to the array.
+ * In this example one contract located at 'build/main.wasm' is deployed with
+ * 0.1 MASSA and an argument 'Test'.
+ *
+ * After all deployments, it terminates the process.
+ */
 (async () => {
   await deploySC(
-    publicApi,
-    deployerAccount,
+    publicApi, // JSON RPC URL
+    deployerAccount, // account deploying the smart contract(s)
     [
       {
-        data: readFileSync(path.join(__dirname, 'build', 'main.wasm')),
-        coins: fromMAS(0.5),
-        args: new Args(),
+        data: readFileSync(path.join(__dirname, 'build', 'main.wasm')), // smart contract bytecode
+        coins: fromMAS(0.1), // coins for deployment
+        args: new Args(), // arguments for deployment
```

Comment:
maybe use noArg here ?

Reply:
it's the same because:
export const NoArg: Args = new Args();

And it adds dependencies("@massalabs/as-types") and import(import { NoArg } from '@massalabs/as-types';) for nothing.

Reply:
ok